COORDINATION.md

```diff
@@ -0,0 +1,103 @@
+| [issue][issues]          | a distinct problem or task                              | what to work on        |
+| [pull request][prs]      | (part of) a solution or fulfillment of a task           | the ongoing work       |
+| [milestone][milestones]  | a distinct feature or a closely related set of problems | what work is towards   |
+| [label][labels]          | a more or less vague category/topic/quality             | attributes of the work |
+| [project board][project] | concurrent human processes with a common dynamic        | how work is done       |
+| [discussion][discussion] | free ranging and open discussion                        | whatever               |
+
+(For more on projects vs. milestones, see
+https://stackoverflow.com/a/47542346/1187277)
+
+[issues]: https://github.com/informalsystems/apalache/issues
+[prs]: https://github.com/informalsystems/apalache/pulls
+[milestones]: https://github.com/informalsystems/apalache/milestones
+[labels]: https://github.com/informalsystems/apalache/issues/labels
+[project]: https://github.com/orgs/informalsystems/projects/30
+[discussion]: https://github.com/informalsystems/apalache/discussions
+
+### Project board
+
+Our project board is the main resource to help with our day-to-day coordination. We use a Kanban-styled continuous workflow board with a high level view of whats being done and some queues for what is up next.
+
+#### Why
+
+The column order in the board represents different points in the timeline:
+
+| Input  | WIP     | Output |
+|--------|---------|--------|
+| Future | Present | Past   |
+
+Our focus should be on the present, and we should look at the future and the
+past to support the present.
+
+For the **present**, we can use the board to help with:
+- Visual feedback of what is being done
+- Capacity management
+- Tactical coordination
+
+For the **future**:
+- Prioritization
+- Aligning expectations
+- Value assessment
+
+For the **past**:
+- Metrics
+- Celebrations
+- Knowledge dissemination
+
+#### How
+
+##### Using the **Next** column
+
+Generally, a backlog is something we cannot control. If a new bug is identified,
+it has to go into some sort of backlog and there is nothing we can do about it.
+Having something so dynamic as our only queue of work is a problem because of
+the overhead it would take to maintain it.
+
+A "Next" column is a subset of our backlog that we can fully control, and the
+idea is for it to hold the tasks that we will work on once we finish what is in
+progress. Therefore, there should be some sort of prioritization criteria to
+help us define which tasks from the backlog should go in this queue. The process
+of deciding which tasks should be on the "Next" column is usually called
+"replenishment".
+
+The Next column needs a size limit since "if everything is a priority, then
+nothing is a priority".
+
+###### How do we prioritize work?
+
+We are currently considering 3 different prioritization schemes:
```

Comment:
From my understanding, scheme 1 and 3 are similar, but in 1 we vote indirectly, by agreeing on the impact and effort of different issues. Is this right, or is there some aspect I am missing?

Reply:
I wouldn't say they are similar, but 3 can be used in combination with any prioritization strategy since every voting person can use their own strategy. 1 and 2 are techniques to prioritizing something and 3 is a suggestion about how to do prioritization in a group. We can opt to use any combination of the three. Maybe this should be clearer in the text.